### ISSUE TITLE
Sortable Table: Enable default browser context menu when right-clicking tables

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#14801]
After removing the custom right-click context menus in #14156, nothing happens when attempting to right-click on a table. Enable the default browser context menu when right-clicking on a table so that the right-click action does not subvert user expectations.

## Acceptance criteria

- [ ] Right clicking on a sortable table raises the default browser context menu
- [ ] Making a select on the sortable table, a user is able to right-click and copy text

## Additional information

It appears that we weren't able to previously copy table selection to the clipboard via right-click. For example: 

- Rancher 2.10 => Home Page => Make selection and right click on table does nothing
- Rancher 2.10 => Local Cluster Explorer => Pods => Make selection and right click on table raises the custom context menu

We definitely want to enable this now in order to maintain user expectations for what happens when right clicking on content.

**Comment by izaac** [comment]
@a-arias should be a quick manual check I recall you've tested this area.

**Comment by a-arias** [comment]
**TESTS** 
Rancher : 2.12-1c40beb39f9125b9bc6864f5fd1f81352158a587-head
UI: master https://github.com/rancher/dashboard/commit/650c7684b8e72bb10100b4434903500ae91190ed

In past versions of Rancher, the custom context menu would override the browser's menu. In the event that a custom menu did not exist, no menu was shown. With this change, the browser's context menu should always be shown on right-click. ✅ 

Example of 2.12 vs 2.10
https://github.com/user-attachments/assets/44a5e85c-adc0-42de-8ff0-0e62f1cbe300

2.12
<img width="1106" height="763" alt="Image" src="https://github.com/user-attachments/assets/2986b46f-11de-4521-a489-d18585cb8941" />

2.10
<img width="946" height="761" alt="Image" src="https://github.com/user-attachments/assets/3ad2933a-69b2-4aed-a6a4-e32f1f531c5f" />
